experiment/kind-e2e.sh

```diff
@@ -71,8 +71,13 @@ build() {
     kind build node-image --type=bazel --kube-root="${PWD}"
 
     # make sure we have e2e requirements
-    #make all WHAT="cmd/kubectl test/e2e/e2e.test vendor/github.com/onsi/ginkgo/ginkgo"
-    bazel build //cmd/kubectl //test/e2e:e2e.test //vendor/github.com/onsi/ginkgo/ginkgo
+    #make all WHAT="cmd/kubectl test/e2e/e2e.test vendor/github.com/onsi/ginkgo/ginkgo cluster/images/conformance/go-runner"
```

Comment:
this script builds with ginkgo directly (or rather kubetest), it shouldn't need to build conformance image stuff?

Reply:
though we're only using it for the sourcing I guess.

really both of these scripts should go away, I at least do not maintain or use these. 

we have a script in the kind repo that we use as a stopgap until kubetest or kubetest2 is decent enough.

Reply:
we were actually going to delete the kind-e2e.sh script previously but found the other script using it, maybe just merge these and remove this one.